assets/app.js

```diff
@@ -1,3 +1,4 @@
+import './bootstrap.js';
```

Comment:
What about the line 19? Isn't it the same?

Reply:
Good catch! I removed this. Thanks!